templates/qemu.conf

```diff
@@ -0,0 +1,19 @@
+#
+# This file is managed with puppet (class ::libvirt::config)
+#    do not edit manually.
+#
+# Master configuration file for the QEMU driver.
+# All settings described here are optional - if omitted, sensible
+# defaults are used.
+#
+# See https://github.com/libvirt/libvirt/blob/master/src/qemu/qemu.conf
+# for possible values.
+<%- @qemu_conf.each do |key,value| -%>
+    <% if ( value.is_a? Array ) -%>
+        <%= key %> = ["<%= value.join(", ") %>"]
+    <% elsif  ( Integer(value) rescue false ) -%>
+        <%= key %> = <%= value %>
+    <% else -%> <%! String !%>
```

Comment:
should be a comment ! dont believe in anything I say ;)

Reply:
ah ok, understood :)